### PR TITLE
Fix missing updated time on migrated issues and comments

### LINKED ISSUE #9741: Wrong meta info at issueComments on github migration
@lunny if I migrate a repo with issues:

comments are generated wich have the correct creation timestamp but update-timestamp is "1970-01-01T01:00:00+01:00"

### PR DESCRIPTION
**Pull request by lunny**
Should fix #9741

### CODE REVIEW

**Review by guillep2k — APPROVED** [review]

File: modules/migrations/github_test.go [hunk 1]
```diff
@@ -157,6 +157,7 @@ func TestGitHubDownloadRepo(t *testing.T) {
 			PosterName: "guillep2k",
 			State:      "closed",
 			Created:    time.Date(2019, 11, 9, 17, 0, 29, 0, time.UTC),
+			Updated:    time.Date(2019, 11, 9, 17, 0, 29, 0, time.UTC),
```
Comment (6543) [inline modules/migrations/github_test.go]:
```suggestion
			Updated:    time.Date(2019, 11, 12, 20, 29, 53, 0, time.UTC),
```

File: modules/migrations/github_test.go [hunk 2]
```diff
@@ -189,6 +190,7 @@ func TestGitHubDownloadRepo(t *testing.T) {
 			PosterName: "mrsdizzie",
 			State:      "closed",
 			Created:    time.Date(2019, 11, 12, 21, 0, 6, 0, time.UTC),
+			Updated:    time.Date(2019, 11, 12, 21, 0, 6, 0, time.UTC),
```
Comment (6543) [inline modules/migrations/github_test.go]:
```suggestion
			Updated:    time.Date(2019, 11, 12, 22, 7, 14, 0, time.UTC),
```

**Review by sapk — DISMISSED** [review]
Need adjustement on test data otherwise LGTM

**Comment by 6543** [comment]
@lunny found second bug:  
`func (g *GithubDownloaderV3) GetPullRequests(page, perPage int) ([]*base.PullRequest, error) {`  
need a patch too around Line  525  [link](https://github.com/go-gitea/gitea/pull/9744/files#diff-7bc639c7c172a3b1e01dc82ddc0d0886L525)

**Review by 6543 — CHANGES_REQUESTED** [review]
as comment

**Comment by 6543** [comment]
@sapk no there is a missing line ...

File: modules/migrations/github_test.go [hunk 3]
```diff
@@ -302,6 +307,7 @@ func TestGitHubDownloadRepo(t *testing.T) {
 			PosterName: "mrsdizzie",
 			State:      "open",
 			Created:    time.Date(2019, 11, 12, 21, 54, 18, 0, time.UTC),
+			Updated:    time.Date(2019, 11, 12, 21, 54, 18, 0, time.UTC),
```
Comment (6543) [inline modules/migrations/github_test.go]:
```suggestion
			Updated:    time.Date(2020, 1, 4, 11, 30, 1, 0, time.UTC),
```

File: modules/migrations/github_test.go [hunk 4]
```diff
@@ -266,6 +270,7 @@ func TestGitHubDownloadRepo(t *testing.T) {
 			PosterName: "mrsdizzie",
 			State:      "closed",
 			Created:    time.Date(2019, 11, 12, 21, 21, 43, 0, time.UTC),
+			Updated:    time.Date(2019, 11, 12, 21, 21, 43, 0, time.UTC),
```
Comment (6543) [inline modules/migrations/github_test.go]:
```suggestion
			Updated:    time.Date(2019, 11, 12, 21, 39, 28, 0, time.UTC),
```

**Comment by 6543** [comment]
```diff
diff --git a/modules/migrations/github.go b/modules/migrations/github.go
index 379af12e7..9183c9318 100644
--- a/modules/migrations/github.go
+++ b/modules/migrations/github.go
@@ -525,6 +525,7 @@ func (g *GithubDownloaderV3) GetPullRequests(page, perPage int) ([]*base.PullReq
                        Milestone:      milestone,
                        State:          *pr.State,
                        Created:        *pr.CreatedAt,
+                       Updated:        *pr.UpdatedAt,
                        Closed:         pr.ClosedAt,
                        Labels:         labels,
                        Merged:         merged,
```

**Comment by techknowlogick** [comment]
Is a new migration needed for these changes (not possible for backport, but what about master)?

**Comment by 6543** [comment]
migration is only needed if we like to fix old records

**Comment by techknowlogick** [comment]
@6543 but this is new column

**Comment by lunny** [comment]
@techknowlogick will add a migration.

**Review by 6543 — APPROVED** [review]

**Comment by 6543** [comment]
@techknowlogick colum already exist in db. But with wrong entrys for migrated issues ...

**Comment by techknowlogick** [comment]
@6543 ah ok. Thanks for pointing that out. I thought this was a different struct.

**Comment by zeripath** [comment]
Make lg-tm work

**Pull request merged**